ci_support/upload_or_check_non_existence.py

```diff
@@ -21,7 +22,12 @@ def built_distribution_already_exists(cli, meta, owner):
 
     """
     distro_name = '{}/{}.tar.bz2'.format(conda.config.subdir, meta.dist())
-    fname = bldpkg_path(meta)
+    try:
+        config = Config()
+        fname = bldpkg_path(meta, config)
+    except TypeError:
+        # conda-build < 2.0.0 takes a single argument
+        fname = bldpkg_path(meta)
```

Comment:
Apparently with a local development version of conda smithy that easy-install.pth is pulling in.  Will submit a fix in a moment.